src/reducers/index.js

```diff
@@ -2,9 +2,11 @@ import { combineReducers } from 'redux';
 import { routerReducer } from 'react-router-redux';
 import query from './query';
 import wikis from './wikis';
+import timeline from './timeline';
 
 export default combineReducers( {
 	router: routerReducer,
 	query,
-	wikis
+	wikis,
+	timeline
```

Comment:
I would change this to `revisions`. That way when we add pages (or whatever) it's not confusing.